styles/google-classroom/catppuccin.user.css

```diff
@@ -0,0 +1,1145 @@
+        @crust: @catppuccin[@@lookup][@crust];
+        @accent-color: @catppuccin[@@lookup][@@accent];
+
+        .XV0XSd .IzuY1c .ZWZruf {
+            background: @mantle;
+        }
+
+        .WbUJNb.VfPpkd-AznF2e-OWXEXe-auswjd .VfPpkd-jY41G-V67aGc,
+        .rvBHac .VfPpkd-AznF2e-wEcVzc-OWXEXe-NowJzb {
+            color: @accent-color;
+            border-color: @accent-color;
+        }
+
+        .WbUJNb .VfPpkd-jY41G-V67aGc {
+            color: @subtext0;
+        }
+
+        /* looks like google made it easy here */
+        .XV0XSd.KkxPLb {
+            --mdc-ripple-color: @accent-color;
+
+            --dt-background: @base;
+            --dt-on-background: @text;
+            --dt-on-background-secondary: @subtext0;
+            --dt-outline: @overlay0;
+            --dt-outline-variant: @overlay1;
+            --dt-on-disabled: rgba(60, 64, 67, 0.38);
+            --dt-disabled: rgba(60, 64, 67, 0.12);
+            --dt-inverse-on-surface: @overlay0;
+            --dt-inverse-surface: @text;
+            --dt-on-surface-variant: @subtext0;
+            --dt-on-surface-secondary: @subtext1;
+            --dt-on-surface: @subtext1;
+            --dt-surface-tint: @surface0;
+            --dt-surface-variant: @surface1;
+            --dt-surface1: @surface1;
+            --dt-surface1-shadow: 0 1px 2px 0 rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);
+            --dt-surface2: @surface1;
+            --dt-surface2-shadow: 0 1px 2px 0 rgba(60, 64, 67, 0.3), 0 2px 6px 2px rgba(60, 64, 67, 0.15);
+            --dt-surface3: @surface1;
+            --dt-surface3-shadow: 0 1px 3px 0 rgba(60, 64, 67, 0.3), 0 4px 8px 3px rgba(60, 64, 67, 0.15);
+            --dt-surface4: @surface1;
+            --dt-surface4-shadow: 0 2px 3px 0 rgba(60, 64, 67, 0.3), 0 6px 10px 4px rgba(60, 64, 67, 0.15);
+            --dt-surface5: @surface1;
+            --dt-surface5-shadow: 0 4px 4px 0 rgba(60, 64, 67, 0.3), 0 8px 12px 6px rgba(60, 64, 67, 0.15);
+            --dt-shadow-elevation-1: 0 1px 2px 0 rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);
+            --dt-shadow-elevation-2: 0 1px 2px 0 rgba(60, 64, 67, 0.3), 0 2px 6px 2px rgba(60, 64, 67, 0.15);
+            --dt-shadow-elevation-3: 0 1px 3px 0 rgba(60, 64, 67, 0.3), 0 4px 8px 3px rgba(60, 64, 67, 0.15);
+            --dt-shadow-elevation-4: 0 2px 3px 0 rgba(60, 64, 67, 0.3), 0 6px 10px 4px rgba(60, 64, 67, 0.15);
+            --dt-shadow-elevation-5: 0 4px 4px 0 rgba(60, 64, 67, 0.3), 0 8px 12px 6px rgba(60, 64, 67, 0.15);
+            --dt-surface: @surface1;
+            --dt-surface-container-lowest: @surface1;
+            --dt-surface-container-low: @surface1;
+            --dt-surface-container: @surface1;
+            --dt-surface-container-high: @surface1;
+            --dt-surface-container-highest: @overlay2;
+            --dt-surface-dim: @surface0;
+            --dt-surface-bright: @surface2;
+            --dt-scrim: rgba(32, 33, 36, 0.6);
```

Comment:
can we remove lines from this group that have hard coded colors or replace them?